pkg/kubectl/cmd/auth/cani.go

```diff
@@ -209,6 +216,10 @@ func (o *CanIOptions) Validate() error {
 				fmt.Fprintf(o.ErrOut, "Warning: resource '%s' is not namespace scoped in group '%s'\n", o.Resource.Resource, o.Resource.Group)
 			}
 		}
+		if !isKnownResourceVerb(o.Verb) {
+			fmt.Fprintf(o.ErrOut, "Warning: verb %s is not a known verb\n", o.Verb)
```

Comment:
to make any whitespace issues clearer:
```suggestion
			fmt.Fprintf(o.ErrOut, "Warning: verb '%s' is not a known verb\n", o.Verb)
```